### DISCUSSION TITLE
[auth] Expo apple authentication with supabase using identityToken?

### DISCUSSION THREAD

**Discussion by xzilja** [supabase/supabase#13565] [Questions]
Hey everyone 👋 

Trying to figure out if I can use expo's native apple authentication library with supabase. In essence implementation to authenticate user in expo with apple looks like this:

```ts
const isAvailable = await AppleAuthentication.isAvailableAsync()
    if (isAvailable) {
      const response = await AppleAuthentication.signInAsync({
        requestedScopes: [
          AppleAuthentication.AppleAuthenticationScope.EMAIL,
          AppleAuthentication.AppleAuthenticationScope.FULL_NAME
        ]
      })
      // TODO: Authenticate with supabase
    }
```

SDK gives us a response where only usable part seems to be `response.identityToken`, in fact this same identity token can be used with other auth libraries like firebase. However I can't figure out how to use it with supabase, so would appreciate any feedback.

Apple sdk used is https://docs.expo.dev/versions/latest/sdk/apple-authentication/

**Comment by Quaza35** [comment]
Hello! currently facing the same issue. did you find a solution?
